### PR TITLE
Update develop branch from CircleCI

### PR DESCRIPTION
**Pull request by hectorerb**
Hi @ajsb85 / @rafaelje 

This PR generates a merge from master to develop branch from CircleCI

### CODE REVIEW

File: ci/scripts/deploy_master.sh [hunk 1]
```diff
@@ -133,6 +133,13 @@ if [[ $GITHUB_COMMIT_MESSAGE != *"ci(release): generate CHANGELOG.md for version
     # Push commit to origin gh-pages branch
     git push origin gh-pages
 
+    # Update develop branch
+    git branch -D develop
+    git fetch origin develop
+    git checkout develop
+    # Merge master on develop
+    git merge $CIRCLE_BRANCH
```
Comment (ajsb85) [inline ci/scripts/deploy_master.sh:141]: I prefer if you write `master` instead the env var.

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Comment by ajsb85** [comment]
LGTM

File: ci/scripts/deploy_master.sh (hunk 1 above)
Comment (hectorerb) [reply ci/scripts/deploy_master.sh:141]: no problem

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Pull request merged**